### DISCUSSION TITLE
How to block additional information

### DISCUSSION THREAD

**Discussion by nbxiglk0** [projectdiscovery/nuclei#3357] [Q&A] [answered]
When I use nuclei as a library it outputs the HTTP messages sent on screen, I have set the defaultOpts' silent to True and debug to Flase and created a NewStandardWriter using the defaultOpts.  
![image](https://user-images.githubusercontent.com/24366795/220868624-fc7be3ac-7eb4-415f-b374-e9de7bb617c5.png)
This is the relevant code setting
```go
	defaultOpts.Silent = True
	defaultOpts.Debug = Flase
	outputWriter, _ := output.NewStandardWriter(defaultOpts)
executerOpts := protocols.ExecuterOptions{
		Output:          outputWriter,
		Options:         defaultOpts,
		Progress:        mockProgress,
		Catalog:         catalog,
		IssuesClient:    reportingClient,
		RateLimiter:     ratelimit.New(context.Background(), 150, time.Second),
		Interactsh:      interactClient,
		HostErrorsCache: cache,
		Colorizer:       aurora.NewAurora(true),
		ResumeCfg:       types.NewResumeCfg(),
	}
```
Do I need to set any parameters if I want to mask this information and just output the results as I did with the realase version?

**Comment by forgedhallpass** [comment, marked as answer]
Have you tried something like:
```go
gologger.DefaultLogger.SetMaxLevel(levels.LevelWarning)

// disable standard logger (ref: https://github.com/golang/go/issues/19895)
log.SetFlags(0)
log.SetOutput(io.Discard)
```

Reference: https://github.com/projectdiscovery/nuclei/blob/96646c8f53d908bc2c009ae658ff4cec7cd85d55/v2/internal/runner/options.go#L247

**Reply by nbxiglk0** [reply]
Thank you for your reply, it was very helpful
